sdk/monitor/Azure.ResourceManager.Monitor/src/autorest.md

```diff
@@ -396,9 +398,9 @@ input-file:
 - https://github.com/Azure/azure-rest-api-specs/blob/a61e5f218a0163431057ed9555eb60c58311792b/specification/monitor/resource-manager/Microsoft.Insights/preview/2018-11-27-preview/vmInsightsOnboarding_API.json
 - https://github.com/Azure/azure-rest-api-specs/blob/a61e5f218a0163431057ed9555eb60c58311792b/specification/monitor/resource-manager/Microsoft.Insights/preview/2021-07-01-preview/privateLinkScopes_API.json
 - https://github.com/Azure/azure-rest-api-specs/blob/a61e5f218a0163431057ed9555eb60c58311792b/specification/monitor/resource-manager/Microsoft.Insights/stable/2020-10-01/activityLogAlerts_API.json
-- https://github.com/Azure/azure-rest-api-specs/blob/a61e5f218a0163431057ed9555eb60c58311792b/specification/monitor/resource-manager/Microsoft.Insights/preview/2021-09-01-preview/dataCollectionEndpoints_API.json
-- https://github.com/Azure/azure-rest-api-specs/blob/a61e5f218a0163431057ed9555eb60c58311792b/specification/monitor/resource-manager/Microsoft.Insights/preview/2021-09-01-preview/dataCollectionRuleAssociations_API.json
-- https://github.com/Azure/azure-rest-api-specs/blob/a61e5f218a0163431057ed9555eb60c58311792b/specification/monitor/resource-manager/Microsoft.Insights/preview/2021-09-01-preview/dataCollectionRules_API.json
+- https://github.com/Azure/azure-rest-api-specs/blob/fce14bf881523f16b14d8ecf325e004465cdcf56/specification/monitor/resource-manager/Microsoft.Insights/stable/2022-06-01/dataCollectionEndpoints_API.json
```

Comment:
We should change the commit id of other files as well. Usually there should be no change on other files